Main/ClientApp/src/Components/Common/Message.tsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Typography } from "@material-ui/core";
+import Center from "./Center";
+
+type MessageProps = {
+  text: string;
+};
+
+export default function Message(props: MessageProps) {
```

Comment:
I don't like the name of this class. Message could mean anything. Consider changing to something like PanelWideMessage